RaisingTheBar.asciidoc

```diff
@@ -50,16 +86,8 @@ Typically you would carry out performance testing in the Test (QA) environment.
 
 === User acceptance testing
 
```

Comment:
Can we add to this section about how we have a number of UAT sessions per Sprint. We don't expect to be able to test all the Features in week 1 of the Sprint, but get the Features through week by week. By having 3 sessions in a Sprint, we have 3 opportunities to get to production rather than only 1.

Reply:
Done